.travis.yml

```diff
@@ -13,12 +13,14 @@ env:
   global:
     - SETUP_XVFB=True
     - CONDA_CHANNELS="astrofrog/label/dev"
-    - CONDA_DEPENDENCIES="astropy qtpy traitlets ipywidgets>=7.0 ipyevents widgetsnbextension pyqt pytest pytest-cov requests nomkl matplotlib beautifulsoup4 lxml jupyterlab flask flask-cors pyopengl"
+    - CONDA_DEPENDENCIES="astropy qtpy traitlets ipywidgets>=7.0 ipyevents widgetsnbextension pyqt pytest pytest-cov requests nomkl matplotlib beautifulsoup4 lxml jupyterlab flask flask-cors pyopengl notebook"
     - PIP_DEPENDENCIES="sphinx-automodapi numpydoc sphinx_rtd_theme pytest-faulthandler codecov"
   matrix:
-    - PYTHON_VERSION=2.7
-    - PYTHON_VERSION=3.5
-    - PYTHON_VERSION=3.6
+		- PYTHON_VERSION=3.6 JUPYTERLAB_VERSION=0.27.0
+		- PYTHON_VERSION=3.6 JUPYTERLAB_VERSION=0.28.0
+		- PYTHON_VERSION=3.6 JUPYTERLAB_VERSION=0.29.0
+		- PYTHON_VERSION=3.6 JUPYTERLAB_VERSION=0.30.0
+		- PYTHON_VERSION=3.6 JUPYTERLAB_VERSION=0.31.0
```

Comment:
Can you change these tabs back to spaces? I wonder if that could be causing issues with Travis?

Reply:
That and Travis fixing the issues they had on their side yesterday worked.